File: 2-python/_python/python-fundamentals/day-0/selection-sort.py
def selection_sort(arr):
	marker = 0
	lowest = 0
	# First, select item in array to work on
	for i in range(len(arr)):
		# Next, loop through all other variables
		for j in range(marker, len(arr)):
			if arr[j] <= lowest:
				arr[j], arr[i] = arr[i], arr[j]
				marker += 1
		lowest += 1
	return arr

print(selection_sort([9,2,7,4,6,0,1,5,3,8]))

File: 3-mean/day-0/js-fundamentals/selection_sort.js
function selectionSort(arr) {
	for (let i = 0; i < arr.length; i++) {
		for (let j = 0; j < arr.length; j++) {
			if (arr[j] > arr[i]) {
				[arr[i], arr[j]] = [arr[j], arr[i]];
			}
		}
	}
	return arr;
}

input = [6, 4, 3, 8, 2, 7, 1]
console.log(selectionSort(input))